src/concepts/segments.md

```diff
@@ -0,0 +1,134 @@
+# Segments
+
+A segment is a group of clients who fit a set of criteria at a point in time. 
+The set of criteria itself can also be referred to as a "segment".
+
+Typically you'll use segments to gain more insight into what is going on, by asking 
+"Regarding the thing I'm interested in, 
+do users in different segments behave differently, 
+and what insights does this give me into the users and the product?" 
+For example, "In this experiment, how do new users react to this feature, 
+and how does this differ from established users?". 
+Or "DAU moved dramatically - 
+is this restricted to users in this particular country (i.e. a segment) 
+where there's an event happening, 
+or is it global and therefore not solely due to that event?"
+
+## Versioning
+
+We are building out our library of segments, 
+and we want room to iterate to improve them in the future. 
+So please quote segments' versions with their names, 
+so that your communication is forwards compatible.
+
+## Current segments
+
+### Regular users v1, Semi-regular users v1, New/irregular users v1
+
+This is a set of three segments. 
+On a given day, every client falls into exactly one of these segments.
+Each client's segment is stored in `telemetry.clients_last_seen.segment_regular_users_v1`.
+(FIXME: or it will be if the [PR](https://github.com/mozilla/bigquery-etl/pull/825) is accepted in a form similar to its current state)
+
+Regular users v1 is defined as 
+clients who have at least two days where they browsed >=5 URIs, 
+in *all four* of the last weeks. 
+Observationally, on any given day this segment seems to contain a large fraction of our users 
+and has exceptionally high retention.
+
+Semi-regular users v1 is defined as 
+clients who browsed >=5 URIs on at least two days in each of *one, two or three* of the last four weeks. 
```

Comment:
This line is confusing.

Reply:
I rephrased - is it better?

Reply:
> *Semi-regular users v1* is the other users: the segment is defined as 
clients who browsed >=5 URIs on at least two days in *some but not all* of the last four weeks. 
This seems to be a smaller segment of our users, and has relatively high retention.

Reply:
What's in your comment looks good.  What's in the branch still seems confusing.

Reply:
In the "Conversation" view, github presents the code as it was when you made the comment (though adds an "Outdated" badge). What's in my comment is in the branch now :)